package.json

```diff
@@ -27,7 +27,7 @@
     "@types/express": "^4.0.33",
     "@types/mime": "0.0.29",
     "@types/mz": "0.0.29",
-    "@types/node": "^4.0.30",
+    "@types/node": "=4.0.30",
```

Comment:
It seems that this package is breaking quite some.other project builds too. Had the same issue with my dom5 PR